lights/light_device/Lamp.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace Lamp_Device{
+    class Lamp
+    {
+        private readonly Socket _client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+        private int PORT;
+
+        public Lamp(int port)
+        {
+            PORT = port;
+        }
+
+        public void StartConnection()
+        {
+            ConnectLoop();
+        }
+
+        public void StartSending()
+        {
+            SendLoop();
+        }
+
+        public void Close()
+        {
+            Exit();
+        }
+
+        private void Exit()
+        {
+            _client.Shutdown(SocketShutdown.Both);
+            _client.Close();
+            Environment.Exit(0);
+        }
+
+        private void SendLoop()
+        {
+            while (true)
+            {
+                Console.WriteLine("Enter a request: ");
+                string request = Console.ReadLine();
+                byte[] buffer = Encoding.ASCII.GetBytes(request);
+                _client.Send(buffer, 0, buffer.Length, SocketFlags.None);
+
+                byte[] receivedBytes = new byte[1024];
```

Comment:
tu też się przyczepię bo te nazwy dla mnie sa troche mylace. dla mnie ten buffer to jest bardziej `receiveBuffer`, albo `receivedData` albo `dataBuffer`, a to ile otrzymaliśmy to bardziej `receivedBytes`